---
string object_id
string object_type
geometry_msgs/Pose pose
bool success
string message